float64 left_vel
float64 right_vel
float64 POLL_TIME
float64 ROBOT_RADIUS
